.circleci/config.yml

```diff
@@ -319,7 +319,7 @@ jobs:
       - show-large-files-and-directories
       - save_cache:
           paths:
-            - fsharp-backend/Build/obj
+            - backend/Build/obj
```

Comment:
Would be worth changing the cache number to prevent CI problems for other branches.

Maybe remove fsharp from the cache name and the CI job name.

Reply:
I removed `fsharp` from the job and cache names - no issue there.

I updated the cache keys:
- from `v5-fsharp-blazor` to `v1-blazor`. This feels totally safe, as I don't think we've used that before.
- from `v15-fsharp-backend` to `v1-backend`. This feels slightly less safe because we've potentially used the key start before. That said, the key also includes the `checksum` after that, so almost definitely just paranoid here.